### PR TITLE
Release notes for 0.6.1

### PR DESCRIPTION
**Pull request by pameladelgado**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,73 @@
...
+**Released 2020-04-01**
+
+New features
+~~~~~~~~~~~~
+
+⭐️ Datasets can be imported from data repositories through the UI
+
+⭐️ Datasets allow uploading file hierarchies in zip format
+
+⭐️ CLI: Datasets metadata is editable. Please see the [Dataset documentation](https://renku-python.readthedocs.io/en/latest/commands.html#module-renku.cli.dataset) for details.
```
Comment (rokroskar) [inline CHANGELOG.rst:15]: this is markdown not rst ;)

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,73 @@
...
+
+⭐️ CLI: enable importing renku datasets
+
+⭐️ CLI: Enable working with data external to the repository [#974](https://github.com/SwissDataScienceCenter/renku-python/pull/974)
```
Comment (rokroskar) [inline CHANGELOG.rst:19]: another markdown link

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,5 +1,73 @@
...
+Individual components
+~~~~~~~~~~~~~~~~~~~~~
+
+For changes to individual components, check:
+
+* renku ui [0.9.1](https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/0.9.1) and [0.9.0](https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/0.9.0
+* renku-python [0.10.2](https://github.com/SwissDataScienceCenter/renku-notebooks/releases/tag/0.10.2)
+* renku-notebooks [0.7.1](https://github.com/SwissDataScienceCenter/renku-notebooks/releases/tag/0.7.1)
+
+* renku-graph [0.55.4 to 0.49.0](https://github.com/SwissDataScienceCenter/renku-graph)
```
Comment (rokroskar) [inline CHANGELOG.rst:61]: markdown

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,5 +1,73 @@
...
+Miscellaneous
+^^^^^^^^^^^^^
+
+- Various improvements on markdown display for collaboration
+
+- Make help channels more visible
+
+- CLI: wildcard support when adding data from git
+
+- Docker images and project templates use Renku [0.10.2](https://github.com/SwissDataScienceCenter/renku-python/releases)
```
Comment (rokroskar) [inline CHANGELOG.rst:39]: markdown

**Comment by pameladelgado** [comment]
Oops, true!
Fixed them now...

**Pull request merged**